### ISSUE TITLE
propulsion rework

### ISSUE THREAD

**Issue by kaki090408** [Eggs-D-Studios/wos-issues#20]
### Guidelines

- [X] I checked for duplicate suggestions
- [X] My suggestion intends to add or improve something

### Version

Development (Unstable)

### Topic

A game mechanic (e.g. a part, a feature, etc)

### Why are you making your suggestion?

it will make the game more balanced, realistic and uhh idk

### What do you propose? What should change, or what should be added?

propulsion rework
what: make larger mass have less top speed and accelerate slower (add acceleration)
why: so that you cant just make a bunker fly at supersonics speeds (larger and heavier ships will be bulkier and slower while smaller ships can be faster and yeah) also missiles can go fast even with small thruster (using thrusters for missiles suck)

**Comment by Pextri** [comment]
I disagree with the idea of limiting the top speed of a craft based on their weight, since they would already be at a disadvantage for changing velocity and acceleration (also idk how it makes it more realistic)
Slower acceleration is also a given regardless as long as the propulsion is given a force value, and parts are given proper weight values. I think there should also be a one use part dedicated for missiles anyways, instead of making it go faster on small weights, to prevent annoying infinite use missiles that follow you to the ends of the earth.

**Comment by Hexcede** [comment]
> make larger mass have less top speed and accelerate slower (add acceleration)

This is sort of a thing in unstable. There is already a top speed for thrusters in stable and unstable. Impulse cannons aren't propulsion, hence why they bypass this top speed.

In unstable your acceleration is affected by the mass of your ship. This needs more work though.
